cisco-nexus-C9332PQ/workflows/deploy-startup-config-boot-images-graph.json

```diff
@@ -18,14 +17,12 @@
                 "options": {
                     "startupConfig": null,
                     "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
-                    "kickstartImage": null,
                     "bootImage": null,
                     "staticDir": "cisco",
-                    "kickstartUri": "{{ api.server }}/{{ options.staticDir }}/{{ options.kickstartImage }}",
                     "bootImageUri": "{{ api.server }}/{{ options.staticDir }}/{{ options.bootImage }}",
                     "commands": [
                         {
-                            "downloadUrl": "/api/2.0/templates/cisco-deploy-config-and-images.py"
+                            "downloadUrl": "/api/2.0/templates/ciscoNexus-deploy-config-and-images.py"
```

Comment:
@lanchongyizu is working on to change the `downloadUrl` to be full path (not relative path), so this will be broken after @lanchongyizu 's code being merged.

- https://github.com/RackHD/on-taskgraph/pull/187
- https://github.com/RackHD/on-skupack/pull/57

This is  the code that actually change the behavior of `downloadUrl`:
- https://github.com/RackHD/on-http/pull/531

Considering @lanchongyizu 's PRs are still under review, need some regress testing. So my suggestion is to merge this PR first, then ask @lanchongyizu to implement some extra changes bases on your PR.

Are you OK? @lanchongyizu @nucklehead ?